### PR TITLE
Add metadata and alternate links

### PR DESCRIPTION
**Pull request by dhiltgen**
Signed-off-by: Daniel Hiltgen <daniel.hiltgen@docker.com>

### CODE REVIEW

**Comment by dhiltgen** [comment]
I didn't want to make too invasive a change in your PR, but I think the pipeline could be done a bit differently to make it cleaner.  I was thinking the image/Makefile could product the offline tar file in `../` someplace, and then wire up the deb/rpm makefiles to use it directly from there instead of the copying model.  Then if a developer does `make image` followed by `make rpm` or `make deb` it would work, and for CI, jenkins can just download the artifact to the top dir any it would work equally well.

I didn't see exactly how the load was working, so the way I have this structured right now with the `docker inspect ...` on the image might go sideways and need a prior `docker load` to make sure it's ready.

File: deb/common/docker-ce.postinst [hunk 1]
```diff
@@ -7,6 +7,7 @@ case "$1" in
 			if ! getent group docker > /dev/null; then
 				groupadd --system docker
 			fi
+                        update-alternatives --install /usr/bin/dockerd dockerd /usr/bin/dockerd-ce 1
```
Comment (dhiltgen) [inline deb/common/docker-ce.postinst:10]: This should verify that `distribution_based_engine.json` on the host says "community" and not "enterprise"

File: rpm/SPECS/docker-ce.spec [hunk 2]
```diff
@@ -82,9 +84,11 @@ fi
 if ! getent group docker > /dev/null; then
     groupadd --system docker
 fi
+update-alternatives --install /usr/bin/dockerd dockerd /usr/bin/dockerd-ce 1
```
Comment (dhiltgen) [inline rpm/SPECS/docker-ce.spec:87]: Same conditional logic here

**Comment by seemethere** [comment]
@dhiltgen Yeah the load was the biggest issue I wanted to avoid, so the way the load the works from the packaging standpoint is basically taking the offline bundle, loading it using `docker load -i offline.tar | sed 's/Loaded image/!d` | cut -d':' -f2- | xargs > engine-image` which ensures loading and that the packaging will use the correct `engine-image`

**Comment by dhiltgen** [comment]
fwiw, another observation is we've got lots and lots of different VERSIONs in these scripts/files and it's really easy to tie ourselves in knots.  This feels like an area where we should try to simplify it down so there's only one `VERSION` variable and the deb/rpm logic then appends the necessary goop on the end of that for the filenames, but the core variable is always `VERSION`   If we did that, then the default could be set to `0.0.0-dev` and be overriden by CI/high-layers in the pipeline, and then the `sed` logic on load would be unnecessary (I think) since the version would "be right" all the time during a build.

**Comment by seemethere** [comment]
@dhiltgen agreed, I think it'd be better if we have a top level makefile with variables that the sub folder level makefiles reference.

I think eventually the `sed` logic may be unnecessary but to get this up and running it'd be best to just keep it as simple as possible.

**Review by seemethere — APPROVED** [review]
LGTM, currently working on making this work on updates, debian packaging was also a bit wonky for me but I can build upon this

**Pull request merged**